admin/tests/unit/routes/authenticated/sessions/list-test.js

```diff
@@ -74,6 +76,30 @@ module('Unit | Route | authenticated/sessions/list', function(hooks) {
         expectedQueryArgs.filter = {
           id: undefined,
           certificationCenterName: 'someName',
+          certificationCenterType: undefined,
+          status: undefined,
+          resultsSentToPrescriberAt: undefined,
+          assignedToSelfOnly: undefined,
+        };
+
+        // when
+        await route.model(params);
+
+        // then
+        sinon.assert.calledWith(route.store.query, 'session', expectedQueryArgs);
```

Comment:
Discussion : on vérifie une query, plutôt un `expect(result).to.equal(expected)` que un `sinon.assert.calledWith(...)` ?